WORKSPACE

```diff
@@ -11,3 +11,9 @@ workspace1()
 load("//envpool:pip.bzl", pip_workspace = "workspace")
 
 pip_workspace()
+
+new_local_repository(
+    name = "cuda",
+    path = "/usr/local/cuda-11.2",
```

Comment:
```suggestion
    path = "/usr/local/cuda",
```
```suggestion
    path = "/usr/local/cuda-11.2",
```

Reply:
I think the issue here is that the "/usr/local/cuda" is a symlink, and bazel can't use. Let me see how this can be made more portable, e.g. let user pass via env var.

Reply:
Using `/usr/local/cuda` works in my side.

Reply:
works in ci machine